docs/user/openstack/control-plane-machine-set.md

```diff
@@ -0,0 +1,324 @@
+
+---
+
+## Example 1: availability zones set to the default
+
+The "default" availability zone `""` means that Nova will schedule instances ignoring availability zones. This is the default setting in OpenShift.
+
+This is the CPMS that the Installer generates for a cluster with name `ocp1-2g2xs`, for which no `zones` have been set in `install-config.yaml`:
+
+```yaml
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: ocp1-2g2xs
+  name: cluster
+  namespace: openshift-machine-api
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: ocp1-2g2xs
+      machine.openshift.io/cluster-api-machine-role: master
+      machine.openshift.io/cluster-api-machine-type: master
+  state: Active
+  strategy: {}
+  template:
+    machineType: machines_v1beta1_machine_openshift_io
+    machines_v1beta1_machine_openshift_io:
+      failureDomains: # <-- Empty property
+        platform: ""
+      metadata:
+        labels:
+          machine.openshift.io/cluster-api-cluster: ocp1-2g2xs
+          machine.openshift.io/cluster-api-machine-role: master
+          machine.openshift.io/cluster-api-machine-type: master
+      spec:
+        lifecycleHooks: {}
+        metadata: {}
+        providerSpec:
+          value: # <-- The OpenStack providerSpec
+            apiVersion: machine.openshift.io/v1alpha1
+            cloudName: openstack
+            cloudsSecret:
+              name: openstack-cloud-credentials
+              namespace: openshift-machine-api
+            flavor: m1.xlarge
+            image: ocp1-2g2xs-rhcos
+            kind: OpenstackProviderSpec
+            metadata:
+              creationTimestamp: null
+            networks:
+            - filter: {}
+              subnets:
+              - filter:
+                  name: ocp1-2g2xs-nodes
+                  tags: openshiftClusterID=ocp1-2g2xs
+            securityGroups:
+            - filter: {}
+              name: ocp1-2g2xs-master
+            serverGroupName: ocp1-2g2xs-master
+            serverMetadata:
+              Name: ocp1-2g2xs-master
+              openshiftClusterID: ocp1-2g2xs
+            tags:
+            - openshiftClusterID=ocp1-2g2xs
+            trunk: true
+            userDataSecret:
+              name: master-user-data
+```
+
+In this case, the `spec.template.machines_v1beta1_machine_openshift_io.failureDomains` stanza does not contain the `openstack` property.
+
+Leaving `failureDomains` without an platform value means: do not substitute the values in the `providerSpec`. In this case, the `providerSpec` does not contain the `availabilityZone` property, because it's implicitly set to the empty string.
```

Comment:
```suggestion
Leaving `failureDomains` without a platform value means: do not substitute the values in the `providerSpec`. In this case, the `providerSpec` does not contain the `availabilityZone` property, because it's implicitly set to the empty string.
```